tests/Unit/Domain/Test_FaceNormal.cpp

```diff
@@ -67,6 +80,120 @@ void check(const Map& map,
+            interface_logical_coordinates(interface_mesh,
+                                          Direction<Dim>(d, Side::Upper)),
+            time, functions_of_time);
+    const InverseJacobian<DataVector, Dim, Frame::Logical, Frame::Inertial>
+        inv_jacobian_lower = logical_to_inertial_map->inv_jacobian(
+            interface_logical_coordinates(interface_mesh,
+                                          Direction<Dim>(d, Side::Lower)),
+            time, functions_of_time);
+
+    for (size_t i = 0; i < Dim; ++i) {
+      CHECK_ITERABLE_APPROX(upper_normal.get(i),
+                            DataVector{inv_jacobian_upper.get(d, i)});
+      CHECK_ITERABLE_APPROX(lower_normal.get(i),
+                            DataVector{-inv_jacobian_lower.get(d, i)});
+    }
+  }
+
+  // Now check the compute items
+  const auto box = db::create<
+      db::AddSimpleTags<Tags::Element<Dim>, Directions<Dim>, Tags::Mesh<Dim>,
+                        Tags::ElementMap<Dim, Frame::Grid>,
+                        CoordinateMaps::Tags::CoordinateMap<Dim, Frame::Grid,
+                                                            Frame::Inertial>,
+                        ::Tags::Time, Tags::FunctionsOfTime>,
+      db::AddComputeTags<
+          Tags::BoundaryDirectionsExterior<Dim>,
+          Tags::InterfaceCompute<Directions<Dim>, Tags::Direction<Dim>>,
+          Tags::InterfaceCompute<Directions<Dim>, Tags::InterfaceMesh<Dim>>,
+          Tags::InterfaceCompute<Tags::BoundaryDirectionsExterior<Dim>,
+                                 Tags::Direction<Dim>>,
+          Tags::InterfaceCompute<Tags::BoundaryDirectionsExterior<Dim>,
+                                 Tags::InterfaceMesh<Dim>>,
+          Tags::InterfaceCompute<
+              Tags::BoundaryDirectionsExterior<Dim>,
+              Tags::UnnormalizedFaceNormalMovingMeshCompute<Dim>>,
+          Tags::InterfaceCompute<
+              Directions<Dim>,
+              Tags::UnnormalizedFaceNormalMovingMeshCompute<Dim>>>>(
+      Element<Dim>(logical_to_grid_map.element_id(), {}), get_directions<Dim>(),
+      Mesh<Dim>{3, Spectral::Basis::Legendre,
+                Spectral::Quadrature::GaussLobatto},
+      ElementMap<Dim, Frame::Grid>(logical_to_grid_map.element_id(),
+                                   logical_to_grid_map.block_map().get_clone()),
+      grid_to_inertial_map->get_clone(), time,
+      clone_unique_ptrs(functions_of_time));
+
+  for (const auto& direction_and_face_normal :
+       db::get<Tags::Interface<Tags::BoundaryDirectionsExterior<Dim>,
+                               Tags::UnnormalizedFaceNormal<Dim>>>(box)) {
+    auto normal = unnormalized_face_normal(
+        interface_mesh, logical_to_grid_map, *grid_to_inertial_map, time,
+        functions_of_time, direction_and_face_normal.first);
+    for (size_t i = 0; i < normal.size(); ++i) {
+      normal.get(i) *= -1.0;
+    }
+
+    CHECK_ITERABLE_APPROX(normal, direction_and_face_normal.second);
```

Comment:
It's unclear to me what's being tested against what. If appropriate, could you name one of these 'expected'? Else, add a comment explaining that this is a consistency, not a correctness, check?